frontend/src/pages/Afterword/index.js

```diff
@@ -73,89 +73,78 @@ const Button = styled(CarbonButton)`
 `;
 
 type Props = {
-  afterwordMessage: string,
   match: any,
   history: *,
-  debrief: boolean,
-  scores: Array<any>,
 };
 
 const clearSessionStorage = async () => await sessionStorage.clear();
 
-class Afterword extends React.Component<Props, State> {
-  async componentDidMount() {
-    getRange(0, 100).map(i => clearInterval(i)); //not the best solution
-    clearSessionStorage().then(() => {
-      persistor.purge();
-    });
-    const { participantUuid } = this.props.match.params;
-    this.props.getDebrief(participantUuid);
-  }
-
-  getHeaders = () => [
+const Afterword = (props: Props) => {
```

Comment:
Only passing match and history into props so maybe we could just do object destructuring here
```
({ match, history }: Props)
```